py/src/braintrust/logger.py

```diff
@@ -958,6 +958,9 @@ def check_updated_param(varname, arg, orig):
 
         conn = None
         if api_key is not None:
+            # If the log_url is set, via BRAINTRUST_API_URL, then use that to login.
```

Comment:
Would a less tricky way to accomplish this be to set `_state.app_url = os.environ.get("BRAINTRUST_API_URL", app_url)` on line 954? Then we wouldn't need to change line 184 either?

Reply:
Yep